README.md

```diff
@@ -88,9 +88,15 @@ Requires admin access. Approves a meme to be publicly viewable.
 
 ### Voting
 
-#### PUT /memes/:meme_id/vote
+#### PUT /memes/:meme_id/:vote_type/vote
```

Comment:
Ideally, we'd pass the 'value' of the React instead of the internal enum representation.

I would suggest changing the route to be `PUT /memes/:meme_id/vote`, and then enforcing the `vote_type` to be in the request body.

The react type should be a string (`like`, `laugh`, etc.), and then the service can interpret that value as the enum, and reject invalid reacts. I think this approach would be more easily extensible, and makes the API a bit cleaner.

Reply:
Changes have been made.